tutorial/practicals/nemo/3_nemo_openmp/README.md

```diff
@@ -251,8 +256,7 @@ poor granularity on MACOS.)
 ## 4. Improving Performance ##
 
 The next section is optional and so, depending on how much time you
-have, you may want to move on to the OpenACC part of this tutorial
-(../4_nemo_openacc). If you are interested but don't have much time
+have, you may want to move on to the [OpenACC part of this tutorial](../4_nemo_openacc). If you are interested but don't have much time
 then example solutions are provided in the
 `parallel_region_omp_trans.py` and
 `general_parallel_region_omp_trans.py` scripts in the `solutions`
```

Comment:
In line 269 `psy.f90` should be `output.f90` now

Reply:
Good spot.